rest_framework/schemas/openapi.py

```diff
@@ -14,6 +15,10 @@
 
 
 class SchemaGenerator(BaseSchemaGenerator):
+    def __init__(self, **kwargs):
+        super(SchemaGenerator, self).__init__(**kwargs)
```

Comment:
as 3.10 is python 3+ only so I guess super().__init__() without explicit calling, SchemaGenerator should be alright.